pkg/client/docker/docker.go

```diff
@@ -93,20 +97,33 @@ func (c *Client) Tags(ctx context.Context, _, repo, image string) ([]api.ImageTa
 				}
 			}
 
+			tag := api.ImageTag{
+				Tag:          result.Name,
+				Timestamp:    timestamp,
+				SHA:          result.Digest, // This isn't _always_ returned
+				OS:           "",
+				Architecture: "",
+			}
+			// Attempt to get OS/Arch, from the Tag Name
+			tag.OS, tag.Architecture = util.OSArchFromTag(result.Name)
```

Comment:
is this necessary? the OS and arch can be retrieved for each image and set as the `tag.OS` and `tag.architecture`, what advantage does setting the parent's OS and arch have?

Reply:
It's a backwards compatibility for when a tag had many different variations of `v0.0.0-linux-arm` and `v0.0.0-linux-amd64` etc